### PR TITLE
store default branch, fix repository added webhook, refactor a lot of github code

### PR DESCRIPTION
**Pull request by factoidforrest**

### CODE REVIEW

File: lunatrace/bsl/docker-compose.yaml [hunk 1]
```diff
@@ -101,8 +101,8 @@ services:
       # TODO: ADD THIS TO THE CDK
       REMOTE_SCHEMA_URL: http://host.docker.internal:3002/graphql/v1
       # COMMENT OUT THESE DIRs IF HASURA WONT START DUE TO MIGRATION OR METADATA ISSUES
-      HASURA_GRAPHQL_METADATA_DIR: /hasura/metadata
-      HASURA_GRAPHQL_MIGRATIONS_DIR: /hasura/migrations
+#      HASURA_GRAPHQL_METADATA_DIR: /hasura/metadata
```
Comment (factoidforrest) [inline lunatrace/bsl/docker-compose.yaml:104]: This is just causing problems on the daily for me, I think we have to drop it @ajvpot  unless anyone feels strongly in which case ill come up with a local override.

Comment (factoidforrest) [reply lunatrace/bsl/docker-compose.yaml:104]: nevermind ill override it locally

**Comment by factoidforrest** [comment]
Chris reviewed live with me

**Review by breadchris — APPROVED** [review]

**Comment by breadchris** [comment]
#466

**Pull request merged**